utils/src/main/kotlin/FileMatcher.kt

```diff
@@ -24,19 +24,19 @@ import java.nio.file.InvalidPathException
 import java.nio.file.Paths
 
 /**
- * A class to determine whether a path resolves to a license file.
+ * A class to determine whether a path is matched by any of the given globs.
  */
-class LicenseFileMatcher(
+class FileMatcher(
     /**
-     * The list of [license files names][fileNames] to consider for matching.
+     * The list of file names to consider for matching.
      */
     val fileNames: List<String>
 ) {
     companion object {
         /**
          * A matcher which uses the default license file names.
          */
-        val DEFAULT_MATCHER = LicenseFileMatcher(
+        val DEFAULT_LICENSE_FILE_MATCHER = FileMatcher(
```

Comment:
How about renaming this to simply `LICENSE_FILE_MATCHER` now, as it's not one of potentially multiple *license* file matchers now anymoer, but *the* license file matcher among the generic file matchers?

Reply:
Done.